_sass/layout/_l-map.scss

```diff
@@ -6,6 +6,10 @@
   right: 0;
   background-color: $bg-color-1;
 
+  .leaflet-bar{
+    border: 0px;
+  }
```

Comment:
If this is to hide that horrible border in tablet... is not working... Check which element has that border...